native-script.rb

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+
+puts "Installing Homebrew (a sudo password prompt may appear)."
+`ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+
+puts "Installing Homebrew-Cask (a sudo password prompt may appear)."
+`brew install caskroom/cask/brew-cask`
+
+puts "Installing Java (a sudo password prompt may appear)."
+`brew cask install java`
+
+puts "Creating NativeScript brew formula."
+File.open("/usr/local/Library/Formula/native-script.rb", "w:utf-8") do |f|
+  f.write DATA.read
+end
+
+puts "Installing NativeScript brew formula."
+`brew install native-script`
+
+__END__
+
+class NativeScript < Formula
+  desc "NativeScript"
+  homepage "https://www.nativescript.org"
+  version "1.1.1"
+  url "https://github.com/NativeScript/nativescript-cli/releases/download/v1.1.1/nativescript.tgz"
+  sha256 "965f652b932acdbd780050d0760715c46a7fa4a621b40be02e348915244f2475"
+
+  depends_on :macos => :yosemite
+  depends_on "pkg-config" => :build
+  depends_on "node"
+  depends_on "ant"
+  depends_on "android-sdk"
+
+  def install
+    ohai "Installing NativeScript."
+    system "/usr/local/bin/npm install -g"
```

Comment:
Yes. it is documented [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#superenv-notes) that npm removes /usr/local/bin before calling my install callback.